src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+
+The [Pointer Events API](https://developers.google.com/web/updates/2016/10/pointer-events)
+provides a unified interface for working with various input options, and is
+especially good for adding stylus support. For supporting both touch and
+keyboard, ensure that you're using the correct semantic elements (anchors,
+buttons, form controls, etc.) and not rebuilding these with non-semantic
+HTML (which is good for accessibility). When including interactions that
+activate on hover, ensure they can activate on click or tap, too.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Provides context for permission requests
+
+When asking permission to use powerful APIs, provide context and ask only
+when the API is needed.
+
+{% endDetailsSummary %}
+
+When asking permission to use powerful APIs, provide context and ask only
+when the API is needed.
+
+#### Why
+
+APIs that trigger a permission prompt, like notifications, geolocation, and
+credentials, are intentionally designed to be disruptive to a user as they
+tend to be related to powerful functionality that requires opt-in. Triggering
+these prompts without additional context, like on page load, makes users
+less likely to accept those permissions and more likely to distrust them
+in the future. Instead, only trigger those prompts after providing an
+in-context rationale to the user for why you need that permission.
+
+#### How
+
+The [Permission UX](https://developers.google.com/web/fundamentals/push-notifications/permission-ux)
+article and UX Planet's
+[The Right Ways to Ask Users for Permissions](https://uxplanet.org/mobile-ux-design-the-right-ways-to-ask-users-for-permissions-6cdd9ab25c27)
+are good resources to understand how to design permission prompts that, while
+focused on mobile, apply to all PWAs.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Follows best practices for healthy code
+
+Keeping your codebase healthy makes it easier to meet your goals and deliver
+new features.
+
+{% endDetailsSummary %}
+
+Keeping your codebase healthy makes it easier to meet your goals and deliver
+new features.
+
+#### Why
+
+There's a lot that goes into building a modern web application. Keeping your
+application up-to-date and your codebase healthy makes it easier for you to
+deliver new features that meet the other goals laid out in this checklist.
+
+#### How
+
+There are a number of high-priority items to check for to ensure a healthy
+codebase: avoiding using libraries with known vulnerabilities, ensuring
+you're not using deprecated APIs, removing web anti-patterns from your
+codebase (like using `document.write()` or having non-passive scroll event
+listeners), and even coding defensively to ensure your PWA doesn't break
+if analytics or other third party libraries fail to load. Consider requiring
+static code analysis, like linting, as well as automated testing, in both
+stable and beta Chrome channels. These techniques can help catch errors
```

Comment:
'in both stable and beta Chrome channels'

Change to:

'in multiple browsers and release channels'

or something like that. Just generalize this beyond Chrome.